std/os/linux.zig

```diff
@@ -259,6 +259,10 @@ pub fn pwrite(fd: i32, buf: [*]const u8, count: usize, offset: usize) usize {
     return syscall4(SYS_pwrite, @bitCast(usize, isize(fd)), @ptrToInt(buf), count, offset);
 }
 
+pub fn copy_file_range(fd_in: i32, off_in: *allowzero isize, fd_out: i32, off_out: *allowzero isize, len: usize, flags: u32) usize {
```

Comment:
Why `allowzero`? are you looking for `?*isize` to make it optional?

Reply:
Thank you @daurnimator, that is exactly what I was looking for.